Assets/Scripts/GameManager.cs

```diff
@@ -8,25 +8,52 @@ public class GameManager : MonoBehaviour
     private float _startingTime = 60f;
 
     [SerializeField]
-    private int _currentLevel = 1;
+    public int _currentLevel;
```

Comment:
if this is gonna be a public property now, some clean up is in order:

1) remove the [SerializedField] attribute
2) remove the underscore
3) i prefer to capitalize public properties to make it clear that they are public. (same with methods -- i try to be consistent about using [pascal case](https://www.theserverside.com/answer/Pascal-case-vs-camel-case-Whats-the-difference) for public method names and camel case for private methods)
4) i prefer keeping properties of the same [access modifiers](https://stackoverflow.com/questions/614818/in-c-what-is-the-difference-between-public-private-protected-and-having-no) together so all the privates are grouped together and all of the publics are grouped together (generally). i tend to put the publics after the privates.